sass/themes/okc/overrides.scss

```diff
@@ -1317,3 +1316,15 @@ div.origin-popup {
 .itinerary-summary-row .itinerary-legs .special-icon.tram .icon {
   vertical-align: middle;
 }
+
+.okc-pdf-download-button {
```

Comment:
Shall we move these definitions up, close to `.route-pattern-select`, to hint at where the respective DOM element is located?